contracts/SportMarkets/SportPositions/SportPositionalMarket.sol

```diff
@@ -392,6 +397,9 @@ contract SportPositionalMarket is OwnedWithInit, ISportPositionalMarket {
     }
 
     function _mint(address minter, uint amount) internal {
+        if (!optionsInitialized) {
```

Comment:
Idk if it matters, but maybe this check needs to go after double chance if statement, so that we are certain that parent market for double chance was initialized first.

Reply:
i didn't see any issue there